src/vnsw/agent/services/arp_handler.cc

```diff
@@ -11,6 +11,7 @@
 #include "services/arp_proto.h"
 #include "services/services_init.h"
 #include "services/services_sandesh.h"
+#include <winnw.h>
```

Comment:
As per our Tuesday discussion, we should avoid breaking compatibility if possible.